apps/ui/src/composables/useUserSkin.ts

```diff
@@ -1,24 +1,29 @@
 type Skin = 'dark' | 'light' | 'none';
 
+const DEFAULT_SKIN: Skin = 'light';
+
 export function useUserSkin() {
   const store = useStorage<Skin>('skin', 'none');
   const currentMode = computed(() =>
-    ['light', 'none'].includes(store.value) ? 'light' : 'dark'
+    [DEFAULT_SKIN, 'none'].includes(store.value) ? DEFAULT_SKIN : 'dark'
   );
 
-  function toggleSkin() {
-    store.value = ['light', 'none'].includes(store.value) ? 'dark' : 'light';
+  function toggleSkin(skin?: Skin) {
+    store.value =
+      skin ||
+      ([DEFAULT_SKIN, 'none'].includes(store.value) ? 'dark' : DEFAULT_SKIN);
```

Comment:
I think I'd separate it as if we pass specific `skin` to set it's not really `toggle` anymore.